assets.html

```diff
@@ -0,0 +1,86 @@
+---
+title: Assets
+layout: default
+permalink: /assets/
+---
+
+<p>Asset files used by Savile are hosted at <code>http://assets.turing.io/</code> and available for public use.</p>
+
+<p>To manage these assets, you will need an AWS account with permissions for the Savile Assets S3 bucket.</p>
+
+<h3 class="s-h3">Images</h3>
+<table class="assets-table">
```

Comment:
I wonder if we can use the `spaced-table` and associated classes to make this table look a little more like the tables on other pages - every other row is dark/light ?

Reply:
Yes! Thanks for the reminder, that would be better